### PR TITLE
Bump angular dependencies to v15

### PR DESCRIPTION
**Pull request by kanidjar**
Today @ngx-matomo is not allowed on Angular 15 (restricted to Angular 14).

Bump Angular dependencies to ^15.0.0

### CODE REVIEW

**Comment by kanidjar** [comment]
hi @EmmanuelRoux, i bumped all angular packages to v15 (ng update @angular-eslint/schematics @angular/cdk  @angular/cli  @angular/core @angular/material) + bumped your @angular/flex-layout to latest version (package is now deprecated).

bumped your tracker + router packages to 4.0.0

File: projects/router/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ngx-matomo/router",
-  "version": "3.2.0",
+  "version": "4.0.0",
```
Comment (EmmanuelRoux) [inline projects/router/package.json]: Version will be bumped automatically when merged, so it should not be manually changed.

File: projects/router/package.json [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
   ],
   "peerDependencies": {
-    "@angular/common": "^14.0.0",
-    "@angular/core": "^14.0.0",
-    "@angular/router": "^14.0.0",
-    "@ngx-matomo/tracker": "3.2.0"
+    "@angular/common": "^15.0.0",
+    "@angular/core": "^15.0.0",
+    "@angular/router": "^15.0.0",
+    "@ngx-matomo/tracker": "3.3.0"
```
Comment (EmmanuelRoux) [inline projects/router/package.json]: Peer dependency to tracker package will be updated automatically when merged, so it should not be manually changed.

File: projects/tracker/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ngx-matomo/tracker",
-  "version": "3.2.0",
+  "version": "4.0.0",
```
Comment (EmmanuelRoux) [inline projects/tracker/package.json]: Version will be bumped automatically when merged, so it should not be manually changed.

File: projects/tracker/schematics/version.ts [hunk 4]
```diff
@@ -1 +1 @@
-export const version = '^3.2.0';
+export const version = '^4.0.0';
```
Comment (EmmanuelRoux) [inline projects/tracker/schematics/version.ts]: Version will be bumped automatically when merged, so it should not be manually changed.

**Review by EmmanuelRoux — CHANGES_REQUESTED** [review]
@kanidjar thank you very much for this PR!

**Review by EmmanuelRoux — COMMENTED** [review]
Could you please run `npm run format` in order to reformat README toc?

**Comment by kanidjar** [comment]
@EmmanuelRoux  done!

**Review by EmmanuelRoux — CHANGES_REQUESTED** [review]
https://github.com/EmmanuelRoux/ngx-matomo/blob/865ef4b3046bc310e5acf0db930f7c89624e3ea6/README.md?plain=1#L42
Almost ready to merge, could you please just either remove this line or replace `Angular 14` with `Angular 15`? (sorry I didn't notice this earlier...)

**Comment by kanidjar** [comment]
@EmmanuelRoux all good!

**Comment by EmmanuelRoux** [comment]
@kanidjar Thank you very much for your contribution!

**Pull request merged**